CBLClient/Apps/CBLTestServer-Dotnet/TestServer/FileLoggingMehtod.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
+using System.Net;
+using JetBrains.Annotations;
+using Couchbase.Lite.Logging;
+
+namespace Couchbase.Lite.Testing
+{
+    public class FileLoggingMehtod
+    {
+        public static void Configure([NotNull] NameValueCollection args,
+                                [NotNull] IReadOnlyDictionary<string, object> postBody,
+                                [NotNull] HttpListenerResponse response)
+        {
+            String log_level = postBody["log_level"].ToString();
+            String directory = postBody["directory"].ToString();
+            int max_rotate_count = (int)postBody["max_rotate_count"];
+            long max_size = (long)postBody["max_size"];
+            bool plain_text = Convert.ToBoolean(postBody["plain_text"]);
+            if (String.IsNullOrEmpty(directory))
+            {
+                directory = System.IO.Path.GetTempPath() + "log_" + (DateTime.UtcNow.Subtract(new DateTime(1970, 1, 1)).TotalSeconds).ToString();
+                Console.WriteLine("File logging configured at: " + directory.ToString());
+            }
+            LogFileConfiguration config = new LogFileConfiguration(directory);
+            if (max_rotate_count > 1)
+            {
+                config.MaxRotateCount = max_rotate_count;
+            }
+            if (max_size > 512000)
```

Comment:
if it is less than 512000, it defaults the value to 512000?

Reply:
yes